src/foam/u2/view/TreeView.js

```diff
@@ -251,21 +251,27 @@ foam.CLASS({
 
   methods: [
     function initE() {
-      var M   = this.ExpressionsSingleton.create();
-      var of  = this.lookup(this.relationship.sourceModel);
+      var M = this.ExpressionsSingleton.create();
```

Comment:
I prefer the code before where the values were all lined up nicely in columns :)